junit-jupiter-params/src/main/java/org/junit/jupiter/params/converter/DefaultArgumentConverter.java

```diff
@@ -246,6 +248,16 @@ public Object convert(String source, Class<?> targetType) throws Exception {
 			return CONVERTERS.get(targetType).apply(source);
 		}
 
+		private static Class<?> toClass(String type) {
+			try {
+				return Class.forName(type);
```

Comment:
Please use `org.junit.platform.commons.util.ReflectionUtils.loadClass(String)` instead of `java.lang.Class.forName(String)`.

Reply:
Please also update the commit message accordingly.

TIA